readme_docker.md

```diff
@@ -0,0 +1,24 @@
+README File For Running Docker Container
+
+The Docker Container contains all of the necessary dependencies and Spacey models already installed
+therefore Gator Miner can be run without downloading everything locally on your machine.
+
+Run the code below to utilize the Docker Container
+
+### Mac:
+
+##### Building:
+Run ```sh docker_build_mac.sh``` for mac
+##### Running:
+
+##### Development command
```

Comment:
I'm not sure what `Development command` is referring to here. I think this header can be more specific

Reply:
Thanks for pointing this out, it is quite badly documented and was originally just jotted down while WIP. I have added this command, merely making the container reflect the local change. I'm leaning toward making the Dockerfile in this repo for development environment setup instead of the actual use case. However, I have not looked into the case if we could just do something similar with an image pulled from the Docker hub. Opening to suggestions, what do you think @BennyWestsyde?